### DISCUSSION TITLE
Adding custom wallet into Rainbow-Kit is not working!

### DISCUSSION THREAD

**Discussion by DragonDev1010** [rainbow-me/rainbowkit#995] [Q&A]
I tried to add KaiKas Wallet which is provided by Klaytn network. I used `connectorsForWallets()` instead of `getDefaultWallets()` in order to import the KaiKas Wallet.

But Connect-Wallet-Modal does not show any wallets.

Following is my code and screenshot of empty Connect-Wallet-Modal.

```
import "@rainbow-me/rainbowkit/styles.css";

import {
  RainbowKitProvider,
  connectorsForWallets
} from "@rainbow-me/rainbowkit";
import {
  injectedWallet,
  metaMaskWallet,
  rainbowWallet,
  walletConnectWallet
} from "@rainbow-me/rainbowkit/wallets";
import { configureChains, createClient, WagmiConfig } from "wagmi";
import { mainnet, polygon, optimism, arbitrum } from "wagmi/chains";
import { alchemyProvider } from "wagmi/providers/alchemy";
import { publicProvider } from "wagmi/providers/public";
import { ConnectBtn } from "./ConnectButton";

export default function App() {
  const { chains, provider } = configureChains(
    [mainnet, polygon, optimism, arbitrum],
    [alchemyProvider({ apiKey: process.env.ALCHEMY_ID }), publicProvider()]
  );

  const { connectors } = connectorsForWallets([
    {
      groupName: "Popular",
      wallets: [
        injectedWallet({ chains }),
        rainbowWallet({ chains }),
        walletConnectWallet({ chains }),
        metaMaskWallet({ chains })
      ]
    }
  ]);

  const wagmiClient = createClient({
    // autoConnect: true,
    connectors,
    provider
  });

  return (
    <WagmiConfig client={wagmiClient}>
      <RainbowKitProvider chains={chains}>
        <ConnectBtn />
      </RainbowKitProvider>
    </WagmiConfig>
  );
}
```

This is the URL of Codesandbox which is containing above code.
[https://codesandbox.io/s/rainbowkit-wallet-david-dzmyjo?file=/src/App.js](url)

This is the screenshot of empty Wallet-Connect-Modal
![Capture](https://user-images.githubusercontent.com/65003082/216158605-f8d656db-4381-48cf-a2d2-94ece6853758.PNG)

Please help me to solve this problem.

**Comment by 0xIvan** [comment]
`connectors` shouldn't be destructured when using `connectorsForWallets` https://www.rainbowkit.com/docs/custom-wallet-list

`const connectors` instead of `const { connectors }`
